src/main/kotlin/ar/edu/unahur/obj2/caralibro/Publicacion.kt

```diff
@@ -1,9 +1,21 @@
 package ar.edu.unahur.obj2.caralibro
 
 import kotlin.math.ceil
+import kotlin.Exception as Exception1
 
 abstract class Publicacion {
+  var cantidadMeGusta = 0
+  var usuariosMeGusta = mutableListOf<Usuario>()
```

Comment:
De paso para evitar repeticiones podrían haber usado un Set
```suggestion
  var usuariosMeGusta = mutableSetOf<Usuario>()
```